deye_daemon.py

```diff
@@ -90,12 +91,29 @@ def __remove_duplicated_reg_ranges(self, reg_ranges: list[SensorRegisterRange])
         return result
 
 
+class setInterval :
+    def __init__(self,interval,action) :
+        self.interval=interval
+        self.action=action
+        self.stopEvent=threading.Event()
+        thread=threading.Thread(target=self.__setInterval)
+        thread.start()
+
+    def __setInterval(self) :
+        nextTime=time.time()+self.interval
+        while not self.stopEvent.wait(nextTime-time.time()) :
+            nextTime+=self.interval
+            self.action()
+
+    def cancel(self, _signum, _frame):
+        self.stopEvent.set()
```

Comment:
Please format this code - at least add spaces. There is no project wide code formatter yet, so be creative. Use black or whatever you prefer.

Reply:
Sorry, usually using black but forgot it here, fixed.